options.go

```
@@ -303,6 +305,68 @@ func WithLeopardGF(enabled bool) Option {
+	for i := range work {
+		if cap(work[i]) < size {
+			work[i] = AllocAligned(1, size)[0]
+		} else {
+			work[i] = work[i][:size]
+		}
+	}
+	return work
+}
+
+func (a *defaultWorkAllocator) Put(work [][]byte) {
+	a.pool.Put(work)
```

golangci-lint (2.11.4)

[error] SA6002: argument should be pointer-like to avoid allocations

(staticcheck)